src/js/pages/ServicesPage.js

```diff
@@ -42,14 +42,23 @@ var ServicesPage = React.createClass({
     this.updateCurrentTab();
   },
 
+  componentDidUpdate: function () {
+    this.updateCurrentTab();
+  },
+
   updateCurrentTab: function () {
     let routes = this.context.router.getCurrentRoutes();
     let currentTab = routes[routes.length - 1].name;
     // `services-page` tab also contains routes for 'services-details'
     if (currentTab === 'services-detail') {
       currentTab = 'services-page';
     }
-    if (currentTab != null) {
+
+    if (currentTab == null) {
```

Comment:
Could we combine this with the `if` statement on line 53? So `if (currentTab === 'services-detail' || currentTab == null) {`...